src/main/target/COLIBRI_RACE/target.h

```diff
@@ -65,6 +65,7 @@
 #define USE_BARO_MS5611
 
 #define MAG
+#define USE_MPU9250_MAG     // Bypass enable
 #define USE_MAG_HMC5883
 #define USE_MAG_AK8975
 
```

Comment:
SPRACINGF3EVO. Read Voltage is 0V without it, and correct value with. I found this line by comparing with the Betaflight version of the file.